doc/ysh-faq.md

```diff
@@ -72,22 +72,24 @@ The rules are subtle, so we avoid inventing new ones.
 
 ## How do I write the equivalent of `echo -e` or `echo -n`?
 
-YSH removed these flags to `echo` because it has good alternatives
+YSH removed these flags to `echo` to solve their interference problem with
+variable substitutions (see next) and double-quoted backslash.
 
-To escape variables, use the statically-parsed string language, rather than
+To echo escape sequences, use a statically-parsed string literal, rather than
```

Comment:
How about "To echo special characters denoted by escape sequences"

Reply:
very good